mkdocs.yml

```
@@ -66,12 +67,17 @@ plugins:
         - secrets: 'secrets/settings.yml'
         - settings: 'study-settings.yml'
       on_undefined: lax
-  - mkpdfs:
-      company: Lausanne University Hospital (CHUV)
-      author: The Axon Lab
-      toc_title: Table of Contents
-      design: design/report.css
-      output_path: pdf/documentation.pdf
+  - mkdocs-jupyter:
+      ignore:
+        - "*.py"
+        - "assets/code/physioconv/.ipynb_checkpoints/Physio to BIDS conversion-checkpoint.ipynb"
+        - "assets/code/physioconv/Physio to BIDS conversion.ipynb"
```

Should we replace the spaces with underscore in the filenames ?
```suggestion
        - "assets/code/physioconv/.ipynb_checkpoints/Physio_to_BIDS_conversion-checkpoint.ipynb"
        - "assets/code/physioconv/Physio_to_BIDS_conversion.ipynb"
```